uportal-war/src/main/java/org/apereo/portal/portlets/lookup/PersonLookupHelperImpl.java

```diff
@@ -440,19 +440,44 @@ protected IAuthorizationPrincipal getPrincipalForUser(final IPerson person) {
     }
 
     /**
+     * Filter the provided set of user attribute names to contain only those the specified principal
+     * has permissions to view.
```

Comment:
This JavaDoc looks like a copy-and-paste of that for `getPermittedAttributes`. Suggest using the JavaDoc to differentiate these methods.